src/systems/userinput/paths.js

```diff
@@ -227,3 +227,27 @@ paths.device.vive.right = {
   },
   pose: `/device/vive/right/pose`
 };
+
+paths.device.wmr = {};
```

Comment:
I think I like how I redid the paths for Oculus Go better, the generic functions kind of lose some of the benefits of making these variables in the first place.